valohai_yaml/objs/merge_mode.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
```

Comment:
This belongs in  `valohai_yaml/enums` or similar; the `objs` package is meant for the objects that make up the YAML files.